cognite/client/data_classes/transformations/common.py

```diff
@@ -140,6 +140,17 @@ def edges(
         """
         return Edges(view=view, instance_space=instance_space, edge_type=edge_type)
 
+    @staticmethod
+    def instances(data_model: Optional[DataModelInfo] = None, instance_space: Optional[str] = None) -> Instances:
+        """
+        Args:
+            dataModel (DataModelInfo): information of the Data Model.
```

Comment:
Please always use snake case 😄 
```suggestion
            data_model (DataModelInfo): information of the Data Model.
```

Reply:
noted 👍